src/coreclr/jit/lowerwasm.cpp

```diff
@@ -110,10 +110,13 @@ GenTree* Lowering::LowerMul(GenTreeOp* mul)
 // Return Value:
 //    The next node to lower (usually nullptr).
 //
+// Notes:
+//    For wasm we handle all this in codegen
+//
 GenTree* Lowering::LowerJTrue(GenTreeOp* jtrue)
 {
-    NYI_WASM("LowerJTrue");
-    return jtrue->gtNext;
+    // TODO-WASM: recognize eqz cases
+    return nullptr;
```

Comment:
Is this null return going to cause an issue anywhere? Or maybe it will cause asserts?

Reply:
The `nullptr` here is the next tree node to process. By convention a `GT_JTRUE` is always the last tree node in a tree (and in the last tree in the block), so there is nothing to process after this.